crates/hulk_webots/src/hardware_interface.rs

```diff
@@ -7,26 +7,31 @@ use std::{
     time::{Duration, SystemTime, UNIX_EPOCH},
 };
 
+use ::hardware::{
+    ActuatorInterface, CameraInterface, IdInterface, MicrophoneInterface, NetworkInterface,
+    SensorInterface, TimeInterface,
+};
 use color_eyre::{
     eyre::{bail, eyre, Error, WrapErr},
     Result,
 };
 use serde::Deserialize;
-use spl_network::endpoint::{Endpoint, Ports};
 use tokio::{
     runtime::{Builder, Runtime},
     select,
 };
 use tokio_util::sync::CancellationToken;
 use types::{
-    hardware::{self, Ids},
+    hardware::Ids,
     messages::{IncomingMessage, OutgoingMessage},
     samples::Samples,
     ycbcr422_image::YCbCr422Image,
     CameraPosition, Joints, Leds, SensorData,
 };
 use webots::Robot;
 
+use spl_network::endpoint::{Endpoint, Ports};
```

Comment:
Why is this separate from the other uses?

Reply:
Presumably because it is from a local crate.

Reply:
From this crates view it is a third-party crate so we should put it into the appropriate group.

Reply:
What would be a first party crate then?
I like to separate uses from registry crates from those from crates belonging to our project.

Reply:
That is a new policy. Until now we formatted completely crate-local. Crates that are not the current crate are considered third-party. Since rustfmt and rust-analyzer handle crates this way, we should not define our own rule here. Please group them together.

Reply:
AFAIK rustfmt and rust-analyzer don't handle import group order at all (by default).
rustfmt sorts within groups and I know there is an option to do some grouping, but its off by default.

Nevertheless I think it makes sense to split off imports belonging to our project from those belonging to third-party crates.
But that's something for after robocup, merged the imports for now for now.

Reply:
Thanks, last remark: I meant that if you quick-fix then it will place them in the correct groups, not if you are formatting. But let's discuss this again after RoboCup. Resolve if read.